docs/tips/19-dangerously-set-inner-html.md

```diff
@@ -6,7 +6,7 @@ permalink: tips/dangerously-set-inner-html.html
 prev: children-undefined.html
 ---
 
-Improper use of the `innerHTML` can open you up to a [cross-site scripting (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting) attack.  Sanitizing user input for display is notoriously error-prone, and failure to properly sanitize is one of the [leading causes of web vulnerabilities](https://owasptop10.googlecode.com/files/OWASP%20Top%2010%20-%202013.pdf) on the internet.
+Improper use of the `innerHTML` can open you up to a [cross-site scripting (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting) attack.  Sanitizing user input for display is notoriously error-prone, and failure to properly sanitize is one of the [leading causes of web vulnerabilities](https://owasptop10.googlecode.com/files/OWASP%20Top%2010%20-%202013.pdf) on the Internet.
```

Comment:
@zpao I have to disagree on this. I think we should take our cue from ICANN and not the Associated Press and definitely not the New York Times. This is particularly apposite given the point being made in the documentation. Cross site scripting is a problem on The One and Only Global Internet In Use By Us All but it may not be a problem on an internet that you own yourself and no one else has access to. See https://en.wikipedia.org/wiki/Capitalization_of_%22Internet%22 

The more substantive problem with this edit is that the link to the XSS PDF appears to be broken. Perhaps it should point to the Wikipedia or an OWASP page on the topic, I'll dig out a couples of refs tomorrow.